packages/typescript-compat/v4.9.x/package.json

```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "@connectrpc/connect-conformance": "*",
     "@connectrpc/example": "*",
-    "@types/node": "22.9.3",
+    "@types/node": "22.10.5",
```

Comment:
Do we really want to bump this version for TS 4.9.x? The package has dist tags for every TS version.
Same for the other compat packages.